nodejs/clients.js

```diff
@@ -1,22 +1,15 @@
-const { readMasterKey, CsfleHelper } = require("./helpers")
+const kms = require("./kms");
+require("dotenv").config();
 
-const localMasterKey = readMasterKey()
-
-const csfleHelper = new CsfleHelper({
-  kmsProviders: {
-    local: {
-      key: localMasterKey
-    }
-  }
-})
+const kmsClient = kms.localCsfleHelper();
 
 async function main() {
   // change this to the base64 encoded data key generated from make-data-key.js
-  let dataKey = null // change this!
+  let dataKey = null; // change this!
```

Comment:
Nitpick: the comment on this line seems redundant given the comment above. I would consolidate the two in one of the locations (either before the line or at the end of it).